src/v2/view-builder/views/email/BaseAuthenticatorEmailView.js

```diff
@@ -82,21 +84,24 @@ const Body = BaseFormWithPolling.extend(Object.assign(
         return;
       }
 
-      if (this.isPollingRateLimitError(error)) {
+      if (this.isRateLimitError(error)) {
         // When polling encounter rate limit error, wait 60 sec for rate limit bucket to reset
         // before polling again & hide error message
-        setTimeout(() => {
-          model.trigger('clearFormError');
```

Comment:
an idea, can we just not clear the errors here? As in, if we have a rate limit error, then don't clear any error messages, doesn't matter if we are polling or not. That way we will keep the error message for 1 minute, AND we don't have to keep a boolean in this view to keep status. At the end, a rate limit error is not something we need to hide the message

Reply:
We would like to hide error if it is not from user interactions, in otherwords, if it is from polling, do not show rate limit error.